site/_posts/2020-12-18-contour_v1110.md

```diff
@@ -0,0 +1,82 @@
+deploying, and managing Contour. The operator extends the functionality of the Kubernetes API to create, configure, and
+manage instances of Contour on behalf of users.
+It builds upon the basic Kubernetes resource and controller concepts, but includes domain-specific knowledge to automate the entire lifecycle of Contour.
+
+In Kubernetes, controllers of the control-plane implement control loops that repeatedly compare the desired state of the cluster to its actual state.
+If the cluster's actual state doesn’t match the desired state, then the controller takes action to fix the problem.
+Contour Operator is a custom Kubernetes controller that uses the `Contour` custom resource (CR) to manage Contour and its dependent components, i.e. the Envoy DaemonSet.
+The `contours.operator.projectcontour.io` Custom Resource Definition (CRD) defines the `Contour` CR. A `Contour` is handled by the Kubernetes API just like built-in objects, including interaction via kubectl and inclusion in role-based access control (RBAC) policies.
+The following example runs an instance of Contour in namespace `projectcontour` with 2 Deployment replicas:
+```
+cat <<EOF | kubectl apply -f -
+apiVersion: operator.projectcontour.io/v1alpha1
+kind: Contour
+metadata:
+  name: contour-sample
+spec: {}
+EOF
+```
+
+No configuration, i.e. `spec`, is provided since `projectcontour` is the default namespace and `2` is the default number
+of replicas for a `Contour`.
+The default parameters of the `Contour` can be verified in the same way as any other Kubernetes resource:
+```
+$ kubectl get contour/contour-sample -o yaml
+apiVersion: operator.projectcontour.io/v1alpha1
+kind: Contour
+metadata:
+  name: contour-sample
+  namespace: default
+...
+spec:
+  namespace:
+    name: projectcontour
+    removeOnDeletion: false
+  replicas: 2
+...
+```
+
+Contour Operator continues to monitor the `Contour` CR and provides status on whether the actual state matches the desired state. The following example indicates the `Contour` is available:
+```
+$ kubectl get contour/contour-sample
+NAME             READY   REASON
+contour-sample   True    ContourAvailable
+```
+
+The `Contour` is now ready to start serving `Ingress` and `HTTPProxy` resources.
+
+Check out the [Getting Started](https://projectcontour.io/getting-started/) guide to learn how to start managing Contour using the operator. Learn more about the planned features of Contour Operator by reviewing the GitHub [Issues](https://github.com/projectcontour/contour-operator/issues).
+
+## XDS Resource `v2` Removed
+
+As mentioned in Contour [1.10](https://projectcontour.io/contour_v1100/#envoy-xds-v3-support) the v2 XDS resource version has been removed from Contour ahead of its removal from Envoy.
+Please see the [XDS Migration Guide](https://projectcontour.io/guides/xds-migration/) for upgrading your instances of Envoy/Contour.
```

Comment:
Mention the External Auth service v2 -> v3 change too, as in the release notes?

Reply:
added a note about that.